### PR TITLE
Minor: Add ParadeDB to the list of users

### PR DESCRIPTION
**Pull request by alamb**
Add another user to the known users list

cc @philippemnoel

### CODE REVIEW

File: docs/source/user-guide/introduction.md [hunk 1]
```diff
@@ -109,6 +109,7 @@ Here are some active projects using DataFusion:
 - [LakeSoul](https://github.com/lakesoul-io/LakeSoul) Open source LakeHouse framework with native IO in Rust.
 - [Lance](https://github.com/lancedb/lance) Modern columnar data format for ML
 - [Parseable](https://github.com/parseablehq/parseable) Log storage and observability platform
+- [ParadeDB](https://github.com/paradedb/paradedb) PostgreSQL for Search
```
Comment (comphead) [inline docs/source/user-guide/introduction.md:112]:
```suggestion
- [ParadeDB](https://github.com/paradedb/paradedb) ParadeDB is an open source ElasticSearch alternative built on Postgres
```

**Review by comphead — APPROVED** [review]
Lgtm thanks @alamb please check the minor

File: docs/source/user-guide/introduction.md (hunk 1 above)
Comment (comphead) [reply docs/source/user-guide/introduction.md:112]: But here is the question, how they use DF if they build on PG?

Comment (philippemnoel) [reply docs/source/user-guide/introduction.md:112]: Quick clarification: We don't implement a foreign data wrapper, but rather a Table Access Method (where we swap the storage engine for Parquet, via `delta-rs`.

Also, I'll make a quick clarification PR on the branding, if that's okay :)

**Comment by philippemnoel** [comment]
Thank you for adding us!

**Pull request merged**